README.md

```diff
@@ -1,5 +1,25 @@
-# DEV1-IMU
+# DEV1 IMU
 
 ## Introduction
 
-*The Inertial Measurement Unit senses the acceleration of the bike and reports any relevant data over CAN.*
\ No newline at end of file
+### DEV1 IMU
+
+The Inertial Measurement Unit measures linear and angular acceleration of the bike using the BNO055 chip. More metrics (gravity, linear and angular velocity, pitch/yaw/roll, and the surrounding magnetic field) can be calculated from the two. The IMU also reports any relevant data over CAN. Such data provides general information to the bike on its orientation and allows other boards to act accordingly for optimal performance. The system is based around the built-in BNO055 chip. The firmware is written for the STM32F334K8U6 microcontroller designed into the DEV1 IMU.
```

Comment:
```suggestion
The Inertial Measurement Unit measures acceleration and position data for the bike using the BNO055 chip. This chip provides a number of metrics, including gravity vector, linear acceleration, angular velocity, Euler angles, and the surrounding magnetic field. The IMU exposes this data over CAN, providing general information to the bike on its orientation and allows other boards to act accordingly for optimal performance. The system is based around the built-in BNO055 chip. The firmware is written for the STM32F334K8U6 microcontroller designed into the DEV1 IMU.
```

I made a few modifications here to make things as accurate as possible. It's important to note that the raw sensors the IMU is using are an accelerometer, magnetometer, and gyroscope. It "fuses" the output of these three sensors to generate the final output data that we read in and use. Basically, this means the chip is doing a bunch of hard math so we don't have to. Importantly, though, the chip does not output linear velocity because with today's technology, it's basically impossible for an IMU to report linear velocity correctly.

On a completely unrelated note, we generally keep line lengths at 80 characters or less. If you're including a hyperlink or something, it's okay to go over, but if you're just putting a big block of text, make sure to break it up between words, so each line doesn't go over 80 characters.